.github/workflows/e2e-tests-on-release.yml

```
@@ -107,6 +107,7 @@ jobs:
       enable_reporting: true
       report_type: RELEASE_CUT
       ref_branch: ${{ needs.validate.outputs.ref_branch }}
+      run_rolling_upgrades: "true"
```

actionlint (1.7.12)

[error] input "run_rolling_upgrades" is typed as string by reusable workflow "./.github/workflows/e2e-tests-playwright.yml". bool value cannot be assigned

(expression)